File: examples/equos-nodejs-integration/src/list-characters.ts
import { client } from "./client.js";

const { total, characters } = await client.characters.listCharacters({
  take: 50,
});

console.log(`${characters.length} of ${total} character(s):`);
for (const c of characters) {
  console.log(`- ${c.id} · ${c.name}`);
}

File: examples/equos-python-integration/src/python_integration/list_characters.py
from python_integration.client import get_client


def list_characters() -> None:
    client = get_client()
    response = client.characters.list(take=50)

    print(f"{len(response.characters)} of {response.total} character(s):")
    for c in response.characters:
        print(f"- {c.id} · {c.name}")


if __name__ == "__main__":
    list_characters()
